src/platforms/web/runtime/modules/style.js

```diff
@@ -9,6 +9,9 @@ const setProp = (el, name, val) => {
   /* istanbul ignore if */
   if (cssVarRE.test(name)) {
     el.style.setProperty(name, val)
+  } else if (name === '-webkit-background-clip') {
```

Comment:
Should we add `&& val === 'text'` too? It only fails on this experimental value.